src/views/AddPeople.vue

```diff
@@ -20,12 +20,12 @@
             <br />
             <input
               class="input"
-              type="number"
+              type="phone"
```

Comment:
You know what's the difference between 'type="number"' and 'type= "phone"', right? When its number you cannot insert any alphabet in there, Its just a number.
Also you cannot just insert anything in type. type is a predefined attribute of html.

Reply:
Yeah you are right! I wanted to use type="tel" instead used phone by mistake.